src/nora/components/IconLink/IconLink.js

```diff
@@ -28,34 +28,30 @@ export const IconLink = ({
   iconName,
   iconClassName,
   iconContainerClassName,
-  linkClassName,
-  linkPosition,
+  textClassName,
```

Comment:
Renamed these props to distinguish between `text` and `icon` since all of these components should be wrapped in a link.